db/migrate/20231031094611_add_id_token_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddIdTokenToUsers < ActiveRecord::Migration[7.1]
```

Comment:
Migration version numbers are different for each file.

Reply:
[48f4ae5](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/48f4ae55aa6a2b473b80efdb367a34189d887d8c)